nbbuild/antsrc/org/netbeans/nbbuild/extlibs/CheckEmbeddedBinaries.java

```diff
@@ -94,9 +97,11 @@ public void execute() throws BuildException {
             throw new BuildException("Could not open " + manifest + ": " + x, x, getLocation());
 
         }
-        try {
-            StringBuilder errorList = new StringBuilder();
-            Files.list(dir.toPath())
+
+        StringBuilder errorList = new StringBuilder();
```

Comment:
The `errorList` can be moved back into the `try` block.

Reply:
Fixed, ty